ports/libsnoretoast/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "libsnoretoast",
+  "version-semver": "0.8.0",
+  "description": "A command line application capable of creating Windows Toast notifications on Windows 8 or later.",
+  "homepage": "https://invent.kde.org/libraries/snoretoast",
+  "supports": "windows",
```

```suggestion
  "supports": "windows & !uwp",
```
Seems this port doesn't support uwp. Could you please help confirm this? If yes, please add `!uwp` to `supports` field.